tests/cpp/operator/mkldnn.cc

```diff
@@ -722,6 +731,27 @@ void TestBinaryOp(const OpAttrs &attrs, VerifyFunc verify_fn) {
   }
 }
 
+TEST(MKLDNN_NDArray, CopyFrom) {
```

Comment:
any modification for this test? Can you move it back to its original place to show the difference more clearly.